src/EditorFeatures/Core.Wpf/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -232,27 +232,31 @@ private async Task<TimeSpan> DownloadFullDatabaseAsync()
                     var (succeeded, delay) = await DownloadFullDatabaseWorkerAsync().ConfigureAwait(false);
                     if (succeeded)
                     {
-                        await _service._progressService.OnDownloadFullDatabaseSucceededAsync(_service._updateCancellationToken).ConfigureAwait(false);
```

Comment:
i investigated this with @heejaechang and this change should not have any impact on hte user experience.  Specifically, we can trace all values that flow into `_service._updateCancellationToken` back to their source and we can already see that in 100% of (non test) cases that only CancellationToken.None is passed here.  So this change will have no effect.

Note:  It is absolutely messed up that we are already only ever passing CancellationToken.None through here through all (non test) cases.  No question about that.

I'm willing to work on a fix for this.  The fix will take one of two forms:

1. we just remove this CT entirely as it's always CancellationToken.None.
2. we actually hook this up properly so that when the OOP server is disconnected we actually set this to to be canceled.

However, in the current state, this PR won't actually accomplish anything.  So i can't sign off on it.